FirebaseRemoteConfig/Tests/Unit/RCNPersonalizationTest.m

```diff
@@ -118,51 +126,97 @@ - (void)testSinglePersonalizationKey {
 
   [_personalization logArmActive:@"key1" config:_configContainer];
 
-  OCMVerify(times(1),
+  OCMVerify(times(2),
             [_analyticsMock logEventWithOrigin:kAnalyticsOriginPersonalization
-                                          name:kAnalyticsPullEvent
+                                          name:[OCMArg isKindOfClass:[NSString class]]
                                     parameters:[OCMArg isKindOfClass:[NSDictionary class]]]);
-  XCTAssertEqual([_fakeLogs count], 1);
+  XCTAssertEqual([_fakeLogs count], 2);
+
+  NSDictionary *logParams = @{
+    kArmKey : @"key1",
+    kArmValue : @"value1",
+    kPersonalizationIdLogKey : @"p13n1",
+    kArmIndexLogKey : @0,
+    kGroup : @"BASELINE"
+  };
+  XCTAssertEqualObjects(_fakeLogs[0], logParams);
 
-  NSDictionary *params = @{kArmKey : @"id1", kArmValue : @"value1"};
-  XCTAssertEqualObjects(_fakeLogs[0], params);
+  NSDictionary *internalLogParams = @{kChoiceIdLogKey : @"id1"};
+  XCTAssertEqualObjects(_fakeLogs[1], internalLogParams);
 }
 
 - (void)testMultiplePersonalizationKeys {
   [_fakeLogs removeAllObjects];
 
   [_personalization logArmActive:@"key1" config:_configContainer];
   [_personalization logArmActive:@"key2" config:_configContainer];
+  [_personalization logArmActive:@"key1" config:_configContainer];
 
-  OCMVerify(times(2),
+  OCMVerify(times(4),
             [_analyticsMock logEventWithOrigin:kAnalyticsOriginPersonalization
-                                          name:kAnalyticsPullEvent
+                                          name:[OCMArg isKindOfClass:[NSString class]]
```

Comment:
IIUC, we no longer assert the event name anywhere. Given a regression in the event name would be bad, I think it's worth asserting.